src/Component/ResumableDropzone/ReactResumableUploader.js

```diff
@@ -19,6 +19,8 @@ export default class ReactResumableUploader extends Component {
         generateUniqueIdentifier: PropTypes.func,
         onFileRemoved: PropTypes.func,
         getFileListCallback: PropTypes.func,
+        uploadToken: PropTypes.string,
```

Comment:
Is this upload token set anywhere?

Reply:
You are right. I set it using the query parameters. Removed.